optimize/self-managed/optimize-deployment/configuration/history-cleanup.md

```diff
@@ -101,7 +95,7 @@ historyCleanup:
 
```

Comment:
can't comment in the right place, but we can also delete the `decisiondata` tab, as this is not supported in C8